### DISCUSSION TITLE
New plugin: Author Fields

### DISCUSSION THREAD

**Discussion by lindsayevans** [payloadcms/payload#2399] [Show and tell]
Hello!

Have just started using Payload recently & have been happy with most things, but I was a bit surprised that it doesn't track the user that created/updated a collection/global.

I spent some time implementing this & decided I'd share it as a plugin (little did I know that the example in the plugins documentation was basically the same thing :p)

After a bit of messing around trying to find the correct hook to use, I've just published v1.0.0 of the Author Fields plugin:

* https://www.npmjs.com/package/@boomworks/payload-plugin-author-fields
* https://github.com/boomworks/payload-plugin-author-fields

Any suggestions welcome :)

**Comment by IRediTOTO** [comment]
I will check this out later

**Comment by matthijs166** [comment]
Nice looking good!
